package.json

```diff
@@ -19,8 +19,7 @@
     "db:up": "pnpm exec drizzle-kit up",
     "db:seed": "node --import tsx --env-file=.env ./scripts/db/seed.ts",
     "db:seed:cleanup": "pnpm run db:seed --cleanup",
-    "test": "pnpm exec jest --runInBand",
-    "test:coverage": "pnpm exec jest --coverage --runInBand",
```

Comment:
I would like to have something similar to this in the new setup. I found the ability to generate test coverage reports extremely useful to spot untested areas of a file. Maybe something like the following should work with our current setup? 

npm install --save-dev nyc
// .nycrc
{
  "extends": "@istanbuljs/nyc-config-typescript",
  "all": true,
  "reporter": ["text", "lcov"],
  "include": ["src/**/*.ts"],
  "exclude": ["**/*.spec.ts"]
}
"scripts": {
  "test:coverage": "nyc -- node --import tsx --env-file=.env --test ./src/**/*.spec.ts"
}

Reply:
great suggestion